card/mod/05_standard/set/rstar/rules.rb

```diff
@@ -166,31 +166,32 @@ def set_fieldset args
     current_set_key = card.new_card? ? Card[:all].cardname.key : card.rule_set_key   # (should have a constant for this?)
     tag = args[:rule_context].rule_user_setting_name
     narrower_rules = []
-    option_list = wrap_each_with :li do
-                    args[:set_options].map do |set_name, state|
-                      
-                      checked    = ( args[:set_selected] == set_name or current_set_key && args[:set_options].length==1 )
-                      is_current = (state == :current)
-                      warning = if narrower_rules.present?
-                                  plural = narrower_rules.size > 1 ? 's' : ''
-                                  "This rule will not have any effect on this card unless you delete the narrower rule#{plural} "+
-                                   "for #{narrower_rules.to_sentence}."
-                                end
-                      if is_current || state == :overwritten
-                         narrower_rules << Card.fetch(set_name).label
-                         narrower_rules.last[0] = narrower_rules.last[0].downcase
-                      end
-                      rule_name  = "#{set_name}+#{tag}"
-                      radio_button( :name, rule_name, :checked=>checked, :warning=>warning ) + %{
-                          <span class="set-label" #{'current-set-label' if is_current }>
-                            #{ card_link set_name, :text=> Card.fetch(set_name).label, :target=>'wagn_set' }
-                            #{'<em>(current)</em>' if is_current }
-                            #{"<em> #{card_link "#{set_name}+#{card.rule_user_setting_name}", :text=>"(overwritten)"}</em>" if state == :overwritten }
-                          </span>
-                         }.html_safe
-                     end
-
-                   end
+    option_list = 
+      wrap_each_with :li do
```

Comment:
I'm not sure about your other point or how it relates.  Any specific code in mind?  The guide is pretty helpful.

The point isn't to suggest that the guide is "right" about any of this.  The point is that if we just agree to follow the guide, then we don't have to debate these style issues point by point.